spec/labor/email_logic_spec.rb

```diff
@@ -3,8 +3,6 @@
 RSpec.describe EmailLogic, type: :labor do
   let(:user) { create(:user) }
 
-  # TODO: improve this test suite, and improve it's speed
```

Comment:
Assuming this was removed because e240f4d speeds up the test sufficiently?

Reply:
I removed it because it's a never ending wish, to have specs improved and sped up :) And the comment doesn't give any specific information...